homeassistant/config_entries.py

```diff
@@ -259,12 +259,20 @@ async def async_setup(
             self.state = ENTRY_STATE_SETUP_RETRY
             wait_time = 2 ** min(tries, 4) * 5
             tries += 1
-            _LOGGER.warning(
-                "Config entry '%s' for %s integration not ready yet. Retrying in %d seconds",
-                self.title,
-                self.domain,
-                wait_time,
-            )
+            if wait_time < 80:
```

Comment:
Maybe just log a warning the first time, and let the rest be debug messages?

Reply:
I like the suggestion. 
Should I word it 

"Config entry '%s' for %s integration not ready yet. Retrying in background"

or something similar ?

Reply:
Sounds good. What do you think @balloob?

Reply:
In the mean time that we wait for Paulus check, I'll updated the code.